Website/app/Contribute.js

```diff
@@ -27,18 +27,24 @@ export default function Contribute( {props} ) {
                 </p>
             </div>
             <div className="md:w-1/2">
-            <h2 className="text-5xl sm:text-4xl font-bold md:mt-20 md:mb-20 md:pl-20 xs:text-center xs:mb-10" >Be a part of Drifty Family!</h2>
-                <div className="grid lg:grid-cols-7 md:grid-cols-5 sm:grid-cols-8 xs:grid-cols-4 space-x-0 gap-y-4 justify-items-center xs:pb-10">
-                    {props.contrib.map((item,index) => {
-                        return <a  href={item.html_url} key={index}><Image width={64} height={64} src={item.avatar_url + ".webp&s=64"} alt={item.login}></Image></a>
+                <h2 className="text-5xl sm:text-4xl my-12 font-bold lg:mt-20 md:mt-10 sm:pt-10 sm:mb-10 md:pl-20 xs:p-5 xs:text-center" >Be a part of Drifty Family!</h2>
```

Comment:
@PriyansuMaurya Please reduce the margin from `20` to `16`.
```suggestion
                <h2 className="text-5xl sm:text-4xl my-12 font-bold lg:mt-16 md:mt-10 sm:pt-10 sm:mb-10 md:pl-20 xs:p-5 xs:text-center" >Be a part of Drifty Family!</h2>
```